compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -102,6 +107,8 @@ def __init__(
         self._time_to_stop = False
         self._kill_event = threading.Event()
 
+        self._heartbeat_period: float = max(5.0, config.heartbeat_period)
```

Comment:
Why the magic number 5?  I looked through interchange.py, config.py, and utils/__init__.py, and heartbeat settings are all over the place.  Hard to make sense of where it is actually set, and what are the overrides.

I feel like the 5 should be shared between single and multi somewhere?  Or one of the existing numbers elsewhere.

Reply:
Five seconds was an arbitrary "don't let users spam the world" choice.  I agree that it should be shared, but I view that as inline with "round 2" of this implementation, and in particular when we coalesce exactly the SU and MU heartbeat codes.

If you'd prefer, I can make that a constant in this module.  But otherwise, I'm inclined to leave it for the time being.

Reply:
Just added `_MINIMUM_HEARTBEAT`.